coremltools/converters/mil/mil/ops/defs/elementwise_unary.py

```diff
@@ -504,7 +504,7 @@ class logical_not(elementwise_unary):
 
     Attributes
     ----------
-    T: fp32
```

Comment:
This seems incorrect. The input and output types both should be `bool` for `logical_not` op.

Reply:
Same for `logical_and` , `logical_or` etc.

Reply:
Sorry, I was confused by `logical_not` under Elementwise Unary, as there is no prototype for it. I missed the Elementwise Binary logical ops because I confused `elementwise_binary_bool_output` with `elementwise_binary_logical`. All these are fixed in the next commit.